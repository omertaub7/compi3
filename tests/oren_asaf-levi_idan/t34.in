int foo(){
  print("Print this");
  return 10;
}

int fooNot(){
  print("Dont print this");
  return 10;
}

void main() {
  int x = 1;
  while(x<=5){
    print("Print me 5 times");
    printi(x);
    x=x+1;
  }
  else{
    print("Im else block");
  }
}