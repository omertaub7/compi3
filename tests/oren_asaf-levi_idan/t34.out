Print me 5 times
1
Print me 5 times
2
Print me 5 times
3
Print me 5 times
4
Print me 5 times
5
Im else block
